tempesta_fw/classifier/frang.c

```diff
@@ -531,6 +577,35 @@ enum {
 	Frang_Req_Done
 };
 
+const char *__state_name_array[] = {
```

Comment:
The array is only used in debug statements. Please make it so it's compiled conditionally.